.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
To cut down on the number of config files, this could be put into `tox.ini`, but I don't have a strong preference.